lib/moonshot/controller.rb

```diff
@@ -67,13 +67,13 @@ def delete
       run_plugins(:post_delete)
     end
 
-    def doctor
+    def doctor(options)
       # @todo use #run_hook when Stack becomes an InfrastructureProvider
       success = true
-      success &&= stack.doctor_hook
-      success &&= run_hook(:build, :doctor)
-      success &&= run_hook(:repo, :doctor)
-      success &&= run_hook(:deploy, :doctor)
+      success &&= stack.doctor_hook(options)
+      success &&= run_hook(:build, :doctor, options)
+      success &&= run_hook(:repo, :doctor, options)
+      success &&= run_hook(:deploy, :doctor, options)
       results = run_plugins(:doctor)
```

Comment:
I think we want plugins to support this, too? I am not sure how that would work though.